src/home.component.tsx

```diff
@@ -1,11 +1,16 @@
 import React from "react";
 import { match, Route } from "react-router";
+import { maybe } from "kremling";
 import HomeDashboard from "./dashboard/home-dashboard.component";
 import PatientSearch from "./patient-search/patient-search.component";
 
 export default function Home(props: HomeProps) {
+  const isPatientSearch = () => {
+    return props.location.pathname.includes("patient-search");
+  };
+
   return (
-    <main className="omrs-main-content">
+    <main className={maybe("omrs-main-content", !isPatientSearch())}>
```

Comment:
Why don't we want omrs-main-content on the patient search page? Now that the navbar is always there, don't we want omrs-main-content to be there?